src/main/kotlin/frc/robot/subsystems/swerve/SwerveDrive.kt

```diff
@@ -178,6 +178,28 @@ class SwerveDrive private constructor
         estimator.resetPosition(pose.rotation, modulePositions, pose)
     }
 
+    private fun isSkidding(): Boolean {
+        var weightedAvg = 0.0
```

Comment:
Of what?

Reply:
87aad34d6ab27dd65877815df2a6db47b6781c3f